ee-feature-pack/galleon-shared/src/main/resources/feature_groups/jgroups-sockets.xml

```diff
@@ -29,7 +24,7 @@
         <param name="multicast-port" value="45688"/>
     </feature>
     <feature spec="socket-binding-group.socket-binding">
-        <param name="socket-binding" value="jgroups-udp-fd"/>
+        <param name="socket-binding" value="jgroups-fd"/>
```

Comment:
@pferraro  FYI I added this to the 'incompatibilities' section of https://github.com/wildfly/wildfly.org/issues/770 (my notes-to-self for the WF 37 release announcement).

@jfdenise Any concerns about this for the cloud feature pack?

Reply:
@pferraro It can break CLI scripts, yaml overrides etc that assume the existing structure. With Galleon we're increasingly moving users toward a provisioning workflow of 'Galleon generates the stock config; you use CLI or yaml to tweak'.

I don't think it's a problem; I'm just trying to mention things like that. It's impossible to catch them all, and probably not worthwhile. A socket-binding though is more likely the kind of thing that people might be tweaking via CLI scripts or yaml.

Reply:
@pferraro I was writing my last comment when you posted yours so I didn't see it. 

I'd note this change in the release announcement regardless, due to the removal of jgroups-tcp-fd. I don't have strong feelings about calling it jgroups-udp-fd vs jgroups-fd. I can vaguely imagine somehow the latter is better for the long term; if so fine. You say 'purely cosmetic' though, so if it's just the 4 extra chars, there are benefits to not changing.

Reply:
@pferraro I also believe this change should be reverted, while I do appreciate the intention to clean this up, this will be problematic for script incompatibility - since the scripts aren't versioned... We have always been strong proponents of using CLI scripts for tuning profiles and this is one of the use cases that I think is relatively common: elect the stack to use, remove the other one and it's bindings that would no longer be used.

Reply:
> this will be problematic for script incompatibility - since the scripts aren't versioned.

Yes, that was Brian's argument above.  Just to say, the removal of FD_SOCK2 and the obsolete jgroups-tcp-fd socket binding already introduces the potential of script compability breakage.

> and this is one of the use cases that I think is relatively common: elect the stack to use, remove the other one and it's bindings that would no longer be used.

In my experience (reviewing user configuration files), this is not actually all that common.  There is very little cost to an unused protocol stack.